casbin/persist/adapter.h

```diff
@@ -33,6 +33,7 @@ void LoadPolicyLine(std::string line, const std::shared_ptr<Model>& model);
 class Adapter {
     public:
 
+        std::stringstream m_policy_stringstream;
```

Comment:
There is some problem in the existing design:  file_path is at adapter level and the corresponding logic which decides to load policy or not as well. 

So I was required to add stringstream member to the adapter basic class. 

I could perform a deep refactoring to make this design better (i.e. to move these members and corresponding logic to the derived classes) but don't think that it is worth it.